src/apiCalls.js

```diff
@@ -0,0 +1,19 @@
+export const getAllCities = () => {
+  return fetch(`https://softwhere-api.herokuapp.com/cities`)
+    .then(response => response.json())
+}
```

Comment:
Let's come back to this and use the checkError function to handle those pesky errors!